internal/datacoord/garbage_collector.go

```diff
@@ -136,13 +137,13 @@ func (gc *garbageCollector) scan() {
 			m++
 			// not found in meta, check last modified time exceeds tolerance duration
 			if time.Since(info.LastModified) > gc.option.missingTolerance {
-				e++
 				// ignore error since it could be cleaned up next time
+				removedKeys = append(removedKeys, info.Key)
```

Comment:
It should be behind 142 lines because the 142 lines maybe failed?

Reply:
> It should be behind 142 lines because the 142 lines maybe failed?

It will retry in the next round if the 142 line failed, so that's ok.